roadmap/roadmap.adoc

```diff
@@ -0,0 +1,158 @@
+include::../shared/issue-footnotes.adoc[]
+
+:m3: ../metametamodel/metametamodel
+:refarch: ../reference-architecture/reference-architecture
+:serialization: ../serialization/serialization
+
+= LionWeb Roadmap, Releases, and History
+:toc:
+:toclevels: 4
+// :sectnums:
+// :sectnumlevels: 4
+
+== Roadmap
+As a volunteer-run Open Source effort, we cannot commit to a timeline.
```

Comment:
I would move that to the end, as the most interesting thing for readers would be the features we are working on, while this is to me an (important) foot-note

Reply:
I agree this is not that important. But without this sentence the next one is hard to understand, and that one should be seen by everybody.